docs/setup-linux.md

```diff
@@ -57,6 +57,11 @@ mkdir -p ~/.config/systemd/user
 ```
 
 Then add this to: `~/.config/systemd/user/kanata.service`
+make sure to update the executable location for sh; you can check with
```

Comment:
I think this would be better to be more specific about the line that needs to change.
```suggestion
Make sure to update the executable location for sh if needed;
i.e. change the line starting with `ExecStart=/usr/bin/sh -c`
You can check with:
```